valhalla/baldr/graphreader.h

```diff
@@ -343,6 +343,12 @@ class GraphReader {
   explicit GraphReader(const boost::property_tree::ptree& pt,
                        std::unique_ptr<tile_getter_t>&& tile_getter = nullptr);
 
+  void SetTileGetterInterrupt(const tile_getter_t::interrupt_t* interrupt) {
```

Comment:
can we name this to SetInterrupt so it looks like the rest of the workers. its ok that only the tile_getter supports being interrupted, but in the future we might want to interrupt any other underlying storage fetcher

Reply:
sure